vuepress/docs/next/tutorials/consume/multitenancy-tutorial.md

```diff
@@ -118,57 +135,111 @@ spec:
+                "kcRealm": "YOUR-TENANT2-REALM",
+                "kcClientId": "quickstart",
+                "kcClientSecret": "YOUR-TENANT2-SECRET",
+                "kcPublicClientId": "entando-web",
+                "kcSecureUris": "",
+                "kcDefaultAuthorizations": "",
+                "dbDriverClassName": "org.postgresql.Driver",
+                "dbUrl": "YOUR-POSTGRESQL-TENANT2-URL",
+                "dbUsername": "postgres",
+                "dbPassword": "xxx",
+                "cdsPublicUrl": "http://cds.YOUR-HOST-URL/YOUR-TENANT2-CODE/",,
+                "cdsPrivateUrl": "http://mt720-cds-tenant2-service.test-mt-720.svc.cluster.local:8080/",
+                "cdsPath": "api/v1",
+                "solrAddress": "http://solr-solrcloud-common.test-mt-720.svc.cluster.local/solr",
+                "solrCore": "TENANT2-CORE-NAME"
+  }
         ]
 ```
 
-2. Apply the ConfigMap
+2. Use kubectl to create a secret for the ConfigMap. Though this step is optional, it is recommended for best practices.
+```
+kubectl -n ent create secret generic tenant-config --from-file=ENTANDO_TENANTS=YOUR-TENANTS-CONFIGMAP
+```
+
 3. Scale down the App Engine deployment (de-app image) to 0
-4. Open the de-app deployment image and edit `envFrom` to point to the new ConfigMap for your secondary tenant under spec.template.spec.containers.
+4. Open the `entando-de-app` deployment and add the environment variable to point to the ConfigMap or to use the K8s Secret:  
+
+4A. to point to the new ConfigMap under spec.template.spec.containers:
+```
 	envFrom:
     	- configMapRef:
-          name: YOUR-TENANT1-CONFIGMAP
-
-### Creating Kubernetes Secrets 
-Entando created the `ENTANDO_TENANTS` environment variables as a Kubernetes Secrets for sensitive tenant configuration parameters. 
-For example:
+          name: YOUR-TENANTS-CONFIGMAP
+```
+4B. If you're using a K8s Secret for the ConfigMap, use this environment variable instead:
 ```
-     envFrom:
-        - secretRef:
-          name: ENTANDO_TENANTS
+envFrom:
+- secretRef:
+    name: tenant-config
+```
+or
+```
+-env:
+   - name: ENTANDO_TENANTS # this name is mandatory, you cannot change it
+     valueFrom:
+        secretKeyRef:
+          key: ENTANDO_TENANTS  # the key used inside the secret 
+          name: tenant-config # the name used for the secret 
+          optional: false
+```
+5. Scale the de-app deployment back up to 1 and check the system and test the new new tenant.
+
+### Tomcat Options
+Entando's multitenancy application uses the Tomcat servlet container and provides a few optional parameters.
+
+**Enabling a Java Agent for Tomcat**
+To use a Java agent with your application, Entando provides a method using the initContainers to prepare the JAR file with information required for the agent. Add the following environment variables to the `entando-de-app-tomcat` deployment to activate the agent and provide the agent code from the JAR file.  
```

Comment:
To use a Java agent with your application, Entando provides a method using the initContainers with a PVC to prepare the JAR file with information required for the agent.